### DISCUSSION TITLE
What color scheme you use ?

### DISCUSSION THREAD

**Discussion by iamrealmarsel** [babel/babel#12999] [Q&A] [answered]
Hi! You have a beautiful color theme in your code examples. Is it your own or publicly available? I would like to use it too in my vscode 🙂

![Screen Shot 2021-03-13 at 7 46 29 PM](https://user-images.githubusercontent.com/20097612/111037401-d1ebf600-8434-11eb-8d46-91e69b13a9b6.png)

**Comment by nicolo-ribaudo** [comment, marked as answer]
Hi!

The theme is Tomorrow Light (https://github.com/chriskempson/tomorrow-theme), but we use a custom background color defined by Highlight.js (`#fdfaeb`).

**Reply by iamrealmarsel** [reply]
Thank you !!
